### PR TITLE
Update to interface with Pool Contracts alpha-30

### PR DESCRIPTION
**Pull request by robsecord**

### CODE REVIEW

File: lib/components/BuilderUI.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useContext, useState } from 'react'
 import { ethers } from 'ethers'
+import { parseInt } from 'lodash'
```
Comment (chuckbergeron) [inline lib/components/BuilderUI.jsx:3]: You can simply use `parseInt()` anywhere in our JS code. But perhaps I'm missing something by not using lodash's version of it?

File: lib/components/BuilderUI.jsx [hunk 2]
```diff
@@ -10,7 +11,7 @@ import { TxMessage } from 'lib/components/TxMessage'
 import { WalletContext } from 'lib/components/WalletContextProvider'
 import { poolToast } from 'lib/utils/poolToast'
 
-
+const now = () => Math.floor((new Date()).getTime() / 1000)
```
Comment (chuckbergeron) [inline lib/components/BuilderUI.jsx:14]: Out of curiousity's sake why are you rounding this down?

File: lib/components/BuilderUI.jsx (hunk 1 above)
Comment (robsecord) [reply lib/components/BuilderUI.jsx:3]: habit..  safely handles discrepancies among older browsers (irrelevant here) as well as iteration safety within lodash iterator functions (also irrelevant here)..  I will remove the import  :)

File: lib/components/BuilderUI.jsx (hunk 2 above)
Comment (robsecord) [reply lib/components/BuilderUI.jsx:14]: Rounding down yields a time in the past, whereas rounding up could yield a time sightly in the future.  Guess it doesn't matter here, just feels more natural

**Pull request merged**